File: Comparativa Python/Ejercicio2-Bucles.py
"""2. Realice un programa que lea de la entrada estándar números hasta que se
introduzca un cero. En ese momento el programa debe terminar y mostrar en la
salida estándar el número de valores mayores que cero leídos."""

contador = 0

num = int(input('Digite un número: '))

while num != 0:
    if num > 0:
        contador += 1
    num = int(input('Digite un número: '))

print('Se digitaron {} números mayores que cero'.format(contador))

File: Ejercicio2-Bucles.cpp
/*2. Realice un programa que lea de la entrada estándar números hasta que se introduzca un cero. En ese momento el programa debe terminar y mostrar en la salida estándar el número de valores mayores que cero leídos.
*/
#include<iostream>
#include<stdlib.h>
using namespace std;
int main(){
    int numero, contador = 0;
    cout<<"Digite un numero: ";
    cin>>numero;
    while (numero!=0){
        if (numero>0){
            contador++;
        }
        cout<<"Digite un numero: ";
        cin>>numero;
    }
    cout<<"Total de numeros mayores que 0: "<<contador<<endl;
    system("pause");
    return 0;
}